src/components/user/edit.vue

```diff
@@ -51,7 +52,8 @@ const props = defineProps({
   id: {
     type: String,
     required: true
-  }
+  },
+  inject: ['config'],
```

Comment:
Unlike Backend, I think ESLint still yells about dangling commas in Frontend. Feel free to turn off that rule!

Reply:
PR at https://github.com/getodk/central-frontend/pull/826